seeds/drinkData.json

```diff
@@ -13,5 +13,15 @@
     "drink_name": "Orange Juice",
```

Comment:
You need to associate liquids with these in the liquid seed so that when you click on the recipe card the liquids populate. Let me know if that doesn't make sense.